apps/api/src/persistence/migrations/migrator.ts

```diff
@@ -45,7 +47,9 @@ export class Migrator {
             .join(`\n`);
     }
 
-    async runAllAvailableMigrations(queryRunner: ICoscradQueryRunner): Promise<void> {
+    async runAllAvailableMigrations(queryRunner: ArangoQueryRunner): Promise<void> {
+        const arangoDataExporter = new ArangoDataExporter(queryRunner);
```

Comment:
Ah- here is the issue. Is it possible to program the `ArangoDataExporter` to the `ICoscradQueryRunner`?  This also begs the question of why we aren't using dependency injection for the data exporter.